Artesian/Artesian.SDK/Service/Query/IVersionedQuery.cs

```diff
@@ -10,7 +10,11 @@ interface IVersionedQuery<T> : IQuery<T>
     {
         T InGranularity(Granularity granularity);
         T ForLastNVersions(int lastN);
-        T ForMUV();
+        T ForMUV(LocalDateTime? versionLimit);
```

Comment:
This is a breaking change.

Any code with ForMUV() will break as the method is no longer available.
Leave ForMUV() with no parameters and make the versionLimit not nullable.
Or make versionLimit? default to NULL.

Reply:
Method defaults to null "public VersionedQuery ForMUV(LocalDateTime? versionLimit = null)" allowing for .ForMUV()  use. Will the interface need to be defaulted to null also?

Reply:
yes because if the user did

```
IActualQuery q = qs.CreateActual();
q.ForMUV();
```

would break. And since that was possible, it must be preserved.

Reply:
Ok , thanks for the clarification.